yarn.lock

```diff
@@ -34670,6 +34778,42 @@ __metadata:
   languageName: node
   linkType: hard
 
+"msw@npm:^2.0.8":
```

Comment:
Sweet, first usages of msw2 - i want to migrate some stuff to it over time, it's SO much nicer